packages/vue-app/template/utils.js

```diff
@@ -273,6 +273,23 @@ export function getQueryDiff(toQuery, fromQuery) {
   return diff
 }
 
+export function normalizeError(err) {
+  let message
+  if (!(err.message || typeof err === 'string')) {
+    try {
+      message = JSON.stringify(err, null, 2)
+    } catch (e) {
+      message = `Non stringified object ${err.constructor.name} `
```

Comment:
@aldarund Combining this type of message with an actual error may cause more confusions for developers (DX) They start thinking, hmm so what's that _non stringified_ thing! What if we just say `[ErrorName]`?